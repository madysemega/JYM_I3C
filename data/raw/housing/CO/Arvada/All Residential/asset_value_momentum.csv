predictions_asset_value_momentum
-0.007181369906102105
-0.013460342661036788
0.03799467809337018
-0.4650449834308968
0.02192107402919056
-0.027332853369286206
0.0383521202986561
-0.09684070716698202
0.020238093149822306
0.0023950071925952434
-0.03986885192775102
-0.05817586212321869
-0.04089413914939317
-0.06664541943324816
-0.09230787532908961
-0.08854379444210564
-0.04025135083162749
-0.10325014632684607
-0.4770139088667306
-0.0931986927931032
-0.09314414711257656
0.020599074032940565
-0.4870807770955764
-0.058624094648647235
-0.17986402265406715
-0.021014883669754826
-0.45198297292313316
