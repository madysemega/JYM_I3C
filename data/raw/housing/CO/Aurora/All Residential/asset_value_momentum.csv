predictions_asset_value_momentum
-3.2143261460015293
-0.36360391930434344
9.520565263124045
0.07891190731904436
-0.010169693640981118
69.47924508672496
-1.9947233002832039
-1.1483370951865917
-4.405208259268148
-7.499572936888962
-0.4814858347368692
-2.020644002161195
-1.0697456611253993
-0.6091772813640715
-1.7299748597396036
0.039677971067748584
-0.20263462240314262
-7.394147701974355
-2.2424830736510537
-2.1329761187752125
-7.973955080049112
-7.408170891743688
-5.115370987402991
-2.8929697575091446
-1.3096540398628387
-5.613860200072518
-1.7500379060344373
