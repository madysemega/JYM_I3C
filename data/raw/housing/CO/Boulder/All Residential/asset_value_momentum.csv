predictions_asset_value_momentum
0.09887746319630954
-0.1549974566195077
0.6687348983962423
-0.05121671205819281
0.32804458967566874
0.16987040764410374
0.47213804386661956
-0.26515167511817656
0.6634666479605361
0.6174404501817115
0.1089251470767386
0.3302820643001603
0.6293813563289142
0.5646091834409516
0.42325825953161705
0.013003788049437168
0.21316637381362752
-0.649101919317074
0.3463739564657117
-0.08792070033371509
-0.13069428591999305
0.2068100222793476
0.38717712168189417
0.6252468667333122
0.18160901167842786
0.4219879613353872
0.08044536198782261
