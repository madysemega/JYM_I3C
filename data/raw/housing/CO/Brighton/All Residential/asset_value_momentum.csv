predictions_asset_value_momentum
-0.07956351361870408
0.06002811774082577
-0.03506352820178974
-0.6651893249097018
0.27664898200442917
0.17153886511939084
0.314254971454836
0.3778948208152226
0.253497917484822
-0.2200338972572028
0.12148411334004705
0.8099592084381768
0.11852392050216266
0.17258078932710255
0.2874938523696518
0.15932096864916476
-0.02351885110921331
0.06835918591178675
0.031191570528844567
0.29655546708877417
0.5920303734501909
0.5452790854532849
-0.26990902582471676
0.6184904960535835
0.46845906211765337
0.29708271557249505
0.564232347726545
