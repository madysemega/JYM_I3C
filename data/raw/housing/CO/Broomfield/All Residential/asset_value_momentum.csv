predictions_asset_value_momentum
-0.016947004272284696
-0.01979107353712566
-0.09047449173188007
0.03575746152752014
-0.008831309129608741
-0.02453943472333985
0.00828836168070886
-0.036039209121041804
-0.03785456639016405
0.04258381741206571
-0.010990642486262536
-0.031767227008751446
-0.020006307578402105
0.024351762935939016
-0.045596047663543124
-0.007877698183725803
0.0806301347420094
-0.0710353115353584
0.04617815402269083
-0.0054630736437953934
-0.006568933404634565
0.041718032480987884
-0.04359606571119657
-0.07102125255060529
-0.03973480940694229
-0.05102728687651047
-0.009215074336900796
