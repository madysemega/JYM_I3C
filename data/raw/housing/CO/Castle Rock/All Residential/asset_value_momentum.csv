predictions_asset_value_momentum
-0.0028742304694781146
-0.17922883788341062
-0.16657853617511287
0.23264981722262865
-0.12866784672269485
0.0037764394621741383
-0.1767755043553447
0.1082191745900757
0.021623628164778884
-0.27976447718625846
-0.18714205784054025
-0.25010030818323586
0.04870892898566313
0.0450394697241191
-0.20062605248315224
0.09583922028379044
0.04751598237361274
0.008297949191957418
-0.3567997567940384
-0.27030676817688937
-0.12063916047786188
0.1077040903415872
-0.10519661470007469
-0.2421018930866539
0.10330003250189493
-0.10692129820101239
-0.12737829498158815
