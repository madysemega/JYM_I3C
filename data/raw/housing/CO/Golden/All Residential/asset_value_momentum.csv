predictions_asset_value_momentum
-0.5047248840559807
-0.0737178114522884
-0.11560518549362823
1.575858469813396
0.07832455759261142
0.00410441774891286
0.7366788656890046
-0.8709601440813833
0.605590953550544
-0.03013494967983545
-0.12253725112841156
1.0138059461755016
-1.1143452026203446
0.23899158389967137
-0.011883040243308956
0.38970084019346113
0.08184971919461896
0.5990021968794998
0.8699226857171536
-0.4529002540695027
0.5936906219561586
0.11914176096789833
-0.5872091765821661
-0.28484538780216107
-0.9467449643041059
-0.0875099021414149
0.20045940637231568
